cmd/soroban-cli/src/config/locator.rs

```diff
@@ -211,6 +211,14 @@ impl Args {
         KeyType::Identity.read_with_global(name, &self.local_config()?)
     }
 
+    pub fn account(&self, account_str: &str) -> Result<Secret, Error> {
```

Comment:
We have `SOROBAN_SECRET_KEY` variable that we are using in stellar keys add to avoid user passing key directly. Should we support the same functionality in `--sign-with-key` for consistency?

Reply:
We have `STELLAR_SIGN_WITH_SECRET` is this good enough? I'm confused how we would reuse `STELLAR_SECRET_KEY` and have it know to use `sign-with-key`.

Reply:
I think the point of that env variable is to not provide the secret key directly to the command, at least my understand of it is that. I assume if we will use SOROBAN_SECRET_KEY in --sign-with-key it should be invoked like this
```
# Can also use sign-with-key, but it accepts a mandatory parameter which would need to be changed
SOROBAN_SECRET_KEY=SKEY stellar contract asset deploy --asset <Asset> --source-account GSOURCE --sign-with-secret 
```

I'm not adamant on this, and we can change it later on